packages/core/src/RepoSync.test.ts

```diff
@@ -276,15 +277,15 @@ describe(`RepoSync`, () => {
     it('should sync local changes made while offline', async () => {
       // remote peer has original state
       let remoteDoc = await remoteRepo.get(documentId)
-      expect(remoteDoc.swallows).toEqual(1)
+      expect(remoteDoc!.swallows).toEqual(1)
```

Comment:
Probably better to assert not-null here

Reply:
👍 c3d6ad6c4a1607cd28297d7b747a989e335f7624